### ISSUE TITLE
Echogenic debris filled stomach

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7080]
**Preferred term label:**
Echogenic debris filled stomach

**Synonyms**

**Definition (free text, please give PubMed ID)**
Sediment within the fetal stomach that is visualized as echogenic debris by fetal sonography.

**Parent term (use hpo.jax.org/app)**
Abnormality of prenatal development or birth HP:0001197
? Better parent term

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ichthyosis prematurity syndrome | 608649

PMID:21856041  (no frequency)

**Your nano-attribution (ORCID)**

**Comment by patt1morgan** [comment]
@carlotarodo, @DominicIliescu - Good day!  We are all slowly trickling back from well needed vacation breaks, hoping to dive back into our prenatal project!  If you could review, and comment on the above term, it would be greatly appreciated. Thank you in advance for your cooperation and help.

**Comment by pnrobinson** [comment]
This can be a normal finding especially in third trimester. However, second-trimester finding is more suspicious. Needs more discussion.

**Comment by patt1morgan** [comment]
Hi Peter,
I found this in the literature, and thought it may help in this term:
Gastric pseudo-mass — In the second and third trimesters, *debris *is
commonly visualized in the fetal stomach as a pseudo-mass consisting of
discrete echogenic areas 4 to 12 mm in diameter  The pseudo-mass is thought
to represent swallowed cells (red blood cells, meconium, fetal skin cells)
that have aggregated due to relatively poor gastric peristaltic activity
early in the second trimester  Gastric pseudo-masses resolve over time and
are not associated with adverse neonatal outcome.

SInce the 'debris' may be sloughed off skin cells, this may explain why it
was seen in your cases of ichthiosis.

**Comment by pnrobinson** [comment]
Thanks, @patt1morgan I added this (even though the clinical utility may be low, it is an abnormal phenotype that people may want to record).
